test/test_pooling.py

```diff
@@ -295,6 +295,7 @@ def test_return_socket_after_reset(self):
 
         self.assertTrue(sock.closed)
         self.assertEqual(0, len(pool.sockets))
+        self.assertGreaterEqual(pool.active_sockets, 0)
```

Comment:
This should be assertEqual 0. Please also add the same assertion for operation_count. 

Finally, let's add the same two assertions inside the with-statement but replace 0 with 1.

Reply:
Done.